src/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -313,29 +367,28 @@ protected function findResponse(IncomingMessage $message): ?ResponseMessage
         if (isset($this->incomingQueue[$actionId])) {
             return $this->incomingQueue[$actionId];
         }
+
         return null;
     }
 
     /**
-     * Dispatchs the incoming message to a handler.
+     * Dispatches the incoming message to a handler.
      *
-     * @param \PAMI\Message\IncomingMessage $message Message to dispatch.
+     * @param \PAMI\Message\Event\EventMessage $message Message to dispatch.
      *
      * @return void
      */
-    protected function dispatch(IncomingMessage $message)
+    protected function dispatch(IncomingMessage $message): void
```

Comment:
Typehint komt niet overeen met de @param

Reply:
Je hebt gelijk, gefixed in de nieuwe commit.